net/seafile-server/Makefile

```diff
@@ -72,7 +72,9 @@ endif
 
 PKG_BUILD_DEPENDS:=vala/host libevhtp
 
-TARGET_CFLAGS += -minterlink-mips16
+ifdef CONFIG_USE_MIPS16
+	TARGET_CFLAGS += -minterlink-mips16
+endif
```

Comment:
It seems the override comes from `$(STAGING_DIR)/mk/python-package.mk`, line 33 and is probably intended to be used when building "real" Python packages. As the Makefile uses the variable `$(PYTHON_PKG_DIR)` I cannot remove the include for this file so please accept this change as is. I've pushed a commit adding a short comment declaring why this "hack" is required.